pyanaconda/payload/__init__.py

```diff
@@ -324,11 +324,17 @@ def __init__(self, data):
         # Additional packages required by installer based on used features
         self.requirements = PayloadRequirements()
 
+        # are mirrors available ?
```

Comment:
Please change this comment to something telling that this is only a placeholder and it will be set in setup method.